docs/src/tutorials/tut_setup.md

```diff
@@ -1,13 +1,320 @@
 # Setting up your simulation from scratch
 
-## Hydrostatic tank
+In this tutorial, we will guide you through the general structure of simulation files.
+We will set up a simulation similar to the example simulation
+[`examples/fluid/hydrostatic_water_column_2d.jl`](https://github.com/trixi-framework/TrixiParticles.jl/blob/main/examples/fluid/hydrostatic_water_column_2d.jl),
+which is one of our simplest example simulations.
+In the second part of this tutorial, we will show how to replace components
+of TrixiParticles.jl by custom implementations from within a simulation file,
+without ever cloning the repository.
 
+For different setups and physics, have a look at [our other example files](@ref examples).
 
+## Resolution
 
+At the beginning of most simulation files, we define the numerical resolution,
+so that it can easily be found and changed.
+First, we import TrixiParticles.jl and
+[OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl), which we will
+use at the very end for the time integration.
+```jldoctest tut_setup; output = false
+using TrixiParticles
+using OrdinaryDiffEq
```

Comment:
This needs:
1) What does numerical resolution mean here?
2) What is the common name that we use?
3) What impact does that have?
4) Concrete values for the example that you are discussing.
...